presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -0,0 +1,106 @@
+package me.androidbox.presentation.agenda.screen
+
+import android.util.Log
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.maxkeppeler.sheets.calendar.CalendarDialog
+import com.maxkeppeler.sheets.calendar.models.CalendarConfig
+import com.maxkeppeler.sheets.calendar.models.CalendarSelection
+import com.maxkeppeler.sheets.calendar.models.CalendarStyle
+import me.androidbox.component.R
+import me.androidbox.component.agenda.AgendaTopBar
+import me.androidbox.component.general.TaskActionButton
+import me.androidbox.component.ui.theme.agendaBackgroundColor
+import me.androidbox.component.ui.theme.backgroundColor
+import me.androidbox.presentation.ui.theme.BusbyTaskyTheme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AgendaScreen(
+    agendaScreenState: AgendaScreenState,
+    agendaScreenEvent: (AgendaScreenEvent) -> Unit,
+    modifier: Modifier = Modifier) {
+
+    CalendarDialog(
+        state = agendaScreenState.calendarState,
+        config = CalendarConfig(
+            style = CalendarStyle.MONTH,
+            monthSelection = true,
+            yearSelection = true
+        ),
+        selection = CalendarSelection.Date { localDate ->
+            agendaScreenEvent(AgendaScreenEvent.OnDateChanged(localDate))
+        }
+    )
```

Comment:
@philipplackner @MKiperszmid Is this the best place to have this calendar?

Reply:
Looks right, yes

Reply:
Looks good, I personally put dialogs at the bottom of the screen composable, since when checking this screen, you likely care more often about the actual screen and not the dialogs. Putting it at the top, you first need to scroll down quite a bit if there are multiple dialogs